### ISSUE TITLE
Opening active users menu crashes browser tab

### ISSUE THREAD

**Issue by AlvaroBrey** [nextcloud/text#3672]
**Describe the bug**
Opening a markdown file and clicking the users menu in the top right produces a cascade of exceptions and the browser tab/window hangs/crashes.

**To Reproduce**
1.  Open any markdown file
2. Click the users menu. No need for the file to be shared or anything else.

I used the `nextcloud:25` docker image to reproduce this.

**Expected behavior**
No tab crash.

**Screenshots**
Tab crash 🤷

**Client details:**
 - OS: Linux (...btw)
 - Browser: Chromium-based only. Firefox doesn't crash. I tested on Brave and Chromium.
 - Version: [e.g. 22]
 - Device: [e.g. iPhone6, desktop]

<details>
<summary>Server details</summary>

**Text app version:** 3.6.0

**Operating system:** Linux (docker)

**Web server:** Apache

**Database:** SQLite

**PHP version:**

**Nextcloud version:** 25.0.2

</details>

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
Nothing relevant

#### Browser log
[toaster.lan-1673890985975.log](https://github.com/nextcloud/text/files/10428281/toaster.lan-1673890985975.log) This exception repeats multiple times per second, crashing the tab.

</details>

**Comment by juliushaertl** [comment]
Do you have any other apps enabled other than the default?

Looks like a leftover of the focus trap issue, so maybe there is still an app using the old version, since text/viewer should have been bumped.

**Comment by AlvaroBrey** [comment]
> Do you have any other apps enabled other than the default?

This was the `nextcloud:25`  (25.0.2) docker image, and I clicked "install recommended apps" during setup. Here's the list of apps that results in:

<details>

```
Enabled:
  - activity: 2.17.0
  - calendar: 4.2.1
  - circles: 25.0.0
  - cloud_federation_api: 1.8.0
  - comments: 1.15.0
  - contacts: 5.0.2
  - contactsinteraction: 1.6.0
  - dashboard: 7.5.0
  - dav: 1.24.0
  - federatedfilesharing: 1.15.0
  - federation: 1.15.0
  - files: 1.20.1
  - files_pdfviewer: 2.6.0
  - files_rightclick: 1.4.0
  - files_sharing: 1.17.0
  - files_trashbin: 1.15.0
  - files_versions: 1.18.0
  - firstrunwizard: 2.14.0
  - logreader: 2.10.0
  - lookup_server_connector: 1.13.0
  - mail: 2.2.2
  - nextcloud_announcements: 1.14.0
  - notifications: 2.13.1
  - oauth2: 1.13.0
  - password_policy: 1.15.0
  - photos: 2.0.1
  - privacy: 1.9.0
  - provisioning_api: 1.15.0
  - recommendations: 1.4.0
  - related_resources: 1.0.3
  - richdocuments: 7.1.0
  - richdocumentscode: 22.5.802
  - serverinfo: 1.15.0
  - settings: 1.7.0
  - sharebymail: 1.15.0
  - spreed: 15.0.2
  - support: 1.8.0
  - survey_client: 1.13.0
  - systemtags: 1.15.0
  - text: 3.6.0
  - theming: 2.0.1
  - twofactor_backupcodes: 1.14.0
  - updatenotification: 1.15.0
  - user_status: 1.5.0
  - viewer: 1.9.0
  - weather_status: 1.5.0
  - workflowengine: 2.7.0
Disabled:
  - admin_audit
  - bruteforcesettings
  - encryption
  - files_external
  - suspicious_login
  - twofactor_totp
  - user_ldap
```

</details>

**Comment by juliushaertl** [comment]
I cannot reproduce this with the 25.0.3 docker anymore, so I guess the app which still had the old focus trap library included has been upgraded in the meantime.

@AlvaroBrey Feel free to reopen if you still see this happening.

**Comment by AlvaroBrey** [comment]
Seems fixed in 25.0.3 indeed. Thanks @juliushaertl !
